src/main/scala/com/netflix/edda/mongo/MongoDatastore.scala

```diff
@@ -365,7 +365,8 @@ class MongoDatastore(val name: String) extends Datastore {
 
   override def remove(queryMap: Map[String, Any])(implicit req: RequestId) {
     try {
-      primary.remove(mapToMongo(queryMap))
+      var opResult = primary.remove(mapToMongo(queryMap, true))
```

Comment:
Sure -- without the literal flag the mapToMongo function was encoding the "$le" into a unicode string.  That caused the query against the mongo database to always return an empty set, breaking the purge policy cleanup and never removing any records.